api/v1beta1/git.go

```diff
@@ -87,4 +87,6 @@ type PushSpec struct {
 	// starting point, if it doesn't already exist.
 	// +required
 	Branch string `json:"branch"`
+	// Force specifies wether to force push the branch to origin
+	Force bool `json:"force"`
```

Comment:
This should be:

```go
	// This flag tells the controller to force push the branch to origin. Defaults to false.
	// +optional
	Force bool `json:"suspend,omitempty"`
```

Reply:
I'm assuming you mean just the `omitempty` part and keeping the key as `force`, not `suspend` - just pushed fix

Reply:
Yes thank you for changing it.